generate-stackbrew-library.sh

```diff
@@ -11,9 +11,12 @@ declare -A debianSuites=(
 	#[10.0]='bullseye'
 )
 
-defaultPhpVersion='php8.3'
+defaultPhpVersion='php8.4'
 declare -A defaultPhpVersions=(
 # releases older than 11 will conservatively stay on 8.2 by default
+	[11.1]='php8.3'
+	[11.0]='php8.3'
+	[10.4]='php8.3'
```

Comment:
Net no change with current releases, just to prepare for 11.2 and 10.5.

Maybe we should consider a minor release for moving the default or just do it anyway?

Reply:
IMO if it's considered fully supported/recommended, we should update the default -- that's sometimes hard to determine though. :disappointed:

The mitigating factor here is that it's really easy for users who experience breakage to explicitly specify their PHP version when they find out that they actually _did_ have a dependency on a specific version of PHP (and now they've learned something important), because we maintain such a generous matrix of supported versions.

Reply:
In lieu of finding an explicit upstream recommendation, if all they document is "support" I'd be comfortable dropping this mapping entirely and having it automatically default to the highest PHP version supported for each Drupal version.